src/styles/components/_input-radio-button-group.scss

```diff
@@ -0,0 +1,134 @@
+.bsds-input-radio {
+  &-button-group {
+    min-width: space(4x);
+  }
+}
+
+.bsds-fieldset-button-group {
+  @extend .bsds-fieldset;
+  flex-flow: row wrap;
+  gap: space(lg);
+
+  .bsds-input-error {
+    width: 100%;
+    margin: 0;
+  }
+
+  .bsds-input-help-text {
+    margin: 0;
+  }
+
+  &[aria-invalid='true'] {
+    .bsds-input-button-group {
+      .bsds-input-radio-button-group {
+        .bsds-input-radio-label-button-group {
+          outline-color: $feedback-error-40;
+
+          &:hover,
+          &:active {
+            outline-color: $action-primary-20;
+            border-color: $action-primary-20;
+          }
+        }
+        /* stylelint-disable scss/selector-nest-combinators */
+        .bsds-input-radio-input {
+          &:checked {
+            + .bsds-input-radio-label-button-group {
+              border-color: $feedback-error-40;
+            }
+          }
+
+          &:focus,
+          &:focus:checked {
+            + .bsds-input-radio-label-button-group {
+              @include focus-invalid;
+            }
+          }
+        }
+      }
+    }
+
+    .bsds-legend-button-group {
+      padding-bottom: 0;
+    }
+  }
+}
+
+.bsds-legend-button-group {
+  @include font-family-base-regular;
+  @include font-size($fs-body-subtle);
+  padding-bottom: space(lg);
+}
+
+.bsds-input-button-group {
+  width: max-content;
+}
+
+.bsds-input-radio-input {
+  + .bsds-input-radio-label-button-group {
+    @include button-default;
+    @include button-small;
+    @include font-family-base-regular;
+    outline: $neutral-40 solid space(xxs);
```

Comment:
can we swap order for outline shorthand (throughout) so it's width style and then color?

i think we might be running into the funky state stuff because we're using outline for the border and focus indicator. let's chat!